docs/reference/ImageGrab.rst

```diff
@@ -26,8 +26,7 @@ or the clipboard to a PIL image memory.
 
         .. versionadded:: 6.2.0
 
-    :param xdisplay: X11 Display address. Pass ``None`` to grab the default system screen.
-                     Pass ``""`` to grab the default X11 screen on Windows or macOS.
+    :param xdisplay: X11 Display address. Pass ``None`` to grab the default system screen. Pass ``""`` to grab the default X11 screen on Windows or macOS.
```

Comment:
👍 

I don't quite understand the logic of why it was doing the odd formatting before and why this fixes it. Do you?